Makefile

```diff
@@ -1,5 +1,5 @@
 # Edit this release and run "make release"
-RELEASE=3.4.4
+RELEASE=3.4.5
```

Comment:
i figured i'd do this here and then i can just run `make release` upon merging :p